test/seq/test_seq.jl

```diff
@@ -1179,4 +1180,70 @@ facts("Translation") do
     @fact_throws translate(rna"ACGUACGNU") # can't translate N
 end
 
+
```

Comment:
I think it would be easier to group all this into a `facts("Parsing") do` block. With this, both `"FASTA Parsing"` and `"FASTAQ parsing"` would become `context` blocks